### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by bfoley13**
Adding pull request template

### CODE REVIEW

**Review by OliverMKing — DISMISSED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. Is it a  breaking change which will impact consuming tool(s).
+
+- [ ] Test A
+- [ ] Test B
```
Comment (jaiveerk) [inline .github/pull_request_template.md:22]: Speaking from personal experience here - would it be useful to add a check of all necessary tests? Namely:
- Unit tests
- E2E locally
- E2E on Github runner
- Potentially E2E on RP but that can't be met by open-source contributors so we can leave that out?

**Review by jaiveerk — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. Is it a  breaking change which will impact consuming tool(s).
```
[the remark this answers is not in the document]
Comment (bfoley13) [reply .github/pull_request_template.md]: fixed

**Review by jaiveerk — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (bfoley13) [reply .github/pull_request_template.md:22]: The thought here is for the person creating the PR to fill this out with what they have done! I think some of these should be required by default so if we want to add a `required` subsection we can later.

Comment (jaiveerk) [reply .github/pull_request_template.md:22]: makes sense!

**Pull request merged**